src/Lykke.Messaging.RabbitMq/RabbitMqTransportFactory.cs

```diff
@@ -14,11 +15,10 @@ public class RabbitMqTransportFactory : ITransportFactory
         private readonly ILogFactory _logFactory;
         private readonly bool m_ShuffleBrokers;
         private readonly TimeSpan? m_AutomaticRecoveryInterval;
+        private readonly Dictionary<TransportInfo, RabbitMqTransport> _transports = new Dictionary<TransportInfo, RabbitMqTransport>();
```

Comment:
Why not ConcurrentDictionary + Lazy? https://andrewlock.net/making-getoradd-on-concurrentdictionary-thread-safe-using-lazy/